app/src/main/java/com/kks/nimblesurveyjetpackcompose/di/NetworkModule.kt

```diff
@@ -91,7 +91,7 @@ interface NetworkModule {
             loggingInterceptor: HttpLoggingInterceptor,
             chuckerInterceptor: ChuckerInterceptor,
             tokenAuthenticator: TokenAuthenticator,
-            @ServiceQualifier accessTokenInterceptor: AccessTokenInterceptor
+            accessTokenInterceptor: AccessTokenInterceptor
```

Comment:
The dependency should be `Interceptor`, and since we're binding `AccessTokenInterceptor` to `Interceptor` already, dagger will provide an `AccessTokenInterceptor` instance anyway.

```suggestion
            interceptor: Interceptor
```